jxl/src/frame/modular.rs

```diff
@@ -347,6 +347,24 @@ impl FullModularImage {
         }
 
         trace!(?section_buffer_indices);
+        for (section, indices) in section_buffer_indices.iter().enumerate() {
```

Comment:
you could put this under a `#[cfg(feature(tracing))]` and skip the underscores (and skip compiling the code if not under tracing)

Reply:
Done.